rgd/utility.py

```diff
@@ -240,3 +241,11 @@ def patch_internal_presign(f: FieldFile):
             f.storage.base_url = original_base_url
         return
     yield
+
+
+def uuid_prefix_filename(instance: Any, filename: str):
+    """Use a variable in settings to add a prefix to the path and keep the random uuid."""
+    prefix = getattr(settings, 'S3_FILE_FIELD_PREFIX', None)
```

Comment:
```suggestion
    prefix = settings.S3_FILE_FIELD_PREFIX
```

I think it would probably be a good idea to add `None` as a default value in `settings.RgdMixin`:
```
    S3_FILE_FIELD_PREFIX = values.Value(default=None, environ=True)
```
That way all customized settings are explicitly called out in `settings.py`.

Reply:
Thanks for the tip!